lib/src/sdk_controller.dart

```diff
@@ -128,33 +233,44 @@ class SitumFlutterSDK {
       case 'onExitedGeofences':
         _onExitGeofences(call.arguments);
         break;
+      case 'navigation.finished':
+        _onNavigationFinished();
+        break;
+      case 'navigation.progress':
+        _onNavigationProgress(call.arguments);
+        break;
+      case 'navigation.oor':
```

Comment:
Porque van con puntos estes cases ? Non deberían de seguir o formato camel case dos demáis ?